workspace/exercise_metadatas_test.go

```diff
@@ -17,7 +17,7 @@ func TestNewSolutions(t *testing.T) {
 		filepath.Join(root, "bravo"),
 		filepath.Join(root, "charlie"),
 	}
-	sx, err := NewSolutions(paths)
+	sx, err := NewExerciseMetadatas(paths)
```

Comment:
I think we need another variable name here, as well (but I don't have any suggestions until we settle on the type name).